easybuild/easyconfigs/p/PETSc/PETSc-3.12.4-foss-2019b.eb

```diff
@@ -26,8 +26,7 @@ builddependencies = [('CMake', '3.15.3')]
 
 dependencies = [
     ('Boost', '1.71.0'),
-    ('METIS', '5.1.0'),
```

Comment:
@migueldiascosta Removing this direct dependency on `METIS` is causing issues with `SLEPc` on top of this `PETSc`:

```
/user/gent/400/vsc40023/eb_phanpyscratch/CO7/skylake-ib/software/PETSc/3.12.4-foss-2019b/lib/libpetsc.so: error: undefined reference to 'METIS_ComputeVertexSeparator'
/user/gent/400/vsc40023/eb_phanpyscratch/CO7/skylake-ib/software/PETSc/3.12.4-foss-2019b/lib/libpetsc.so: error: undefined reference to 'METIS_NodeND'
collect2: error: ld returned 1 exit status
```

Reply:
`METIS` should be loaded through the dependency on `MUMPS`, isn't it the case?

Reply:
I can see the errors of `ld` regarding the undefined references on `METIS` in my successful test, but EB ignores those errors. These seems more of a bug in `petsc` easyblock than in this easyconfig.

Reply:
fixed in #9891